dist/index.html

```diff
@@ -37,7 +37,7 @@
         </div>
         <div class="row top-description">
             <div class="col-md-12">
-                <p>Dream it. Hack it. Ship it.<br>500 hackers. 36h.<br>Barcelona, Spain | 7 - 9 Oct</p>
+                <p>Dream it. Hack it. Ship it.<br>500 hackers. 36h.</p>
```

Comment:
The date is already on the header/title, so no need to repeat info on the same screen.